### ISSUE TITLE
Clipboard warning

### ISSUE THREAD

**Issue by kring** [CesiumGS/cesium-unreal#215]
After #213 there's a compile-time warning:

```
1>C:/cesium/cesium-unreal-demo-2/Plugins/cesium-unreal/Source/CesiumEditor/IonLoginPanel.cpp(157): warning C4996: 'FGenericPlatformMisc::ClipboardCopy': FPlatformMisc::ClipboardCopy() has been superseded by FPlatformApplicationMisc::ClipboardCopy() Please update your code to the new API before upgrading to the next release, otherwise your project will no longer compile.
```
